examples/rpc-db/src/main.rs

```diff
@@ -1,71 +1,44 @@
-//! Example illustrating how to run the ETH JSON RPC API as standalone over a DB file.
-//!
-//! Run with
-//!
-//! ```not_rust
-//! cargo run -p rpc-db
-//! ```
-//!
-//! This installs an additional RPC method `myrpcExt_customMethod` that can queried via [cast](https://github.com/foundry-rs/foundry)
-//!
-//! ```sh
-//! cast rpc myrpcExt_customMethod
-//! ```
-
 use std::{path::Path, sync::Arc};
-
 use reth::{
     api::NodeTypesWithDBAdapter,
     beacon_consensus::EthBeaconConsensus,
-    providers::{
-        providers::{BlockchainProvider, StaticFileProvider},
-        ProviderFactory,
-    },
+    providers::{providers::{BlockchainProvider, StaticFileProvider}, ProviderFactory},
     rpc::eth::EthApi,
     utils::open_db_read_only,
 };
 use reth_chainspec::ChainSpecBuilder;
 use reth_db::{mdbx::DatabaseArguments, ClientVersion, DatabaseEnv};
-
-// Bringing up the RPC
-use reth::rpc::builder::{
-    RethRpcModule, RpcModuleBuilder, RpcServerConfig, TransportRpcModuleConfig,
-};
-// Configuring the network parts, ideally also wouldn't need to think about this.
+use reth::rpc::builder::{RethRpcModule, RpcModuleBuilder, RpcServerConfig, TransportRpcModuleConfig};
 use myrpc_ext::{MyRpcExt, MyRpcExtApiServer};
 use reth::{blockchain_tree::noop::NoopBlockchainTree, tasks::TokioTaskExecutor};
-use reth_node_ethereum::{
-    node::EthereumEngineValidator, EthEvmConfig, EthExecutorProvider, EthereumNode,
-};
```

Comment:
i find these comments explaining why each import is required, to be useful, let's add them back?